R/database.R

```diff
@@ -2158,8 +2151,19 @@ get_ssm_by_region = function(chromosome,
           print("NAMES:")
           print(maf_columns)
         }
-        muts_region = vroom::vroom(I(muts), col_types = paste(maf_column_types,collapse=""),
+        if(length(muts)==0){
+          maf_types_sep = str_split(maf_column_types,pattern="")[[1]] %>% 
+            str_replace_all("c","character") %>%
+            str_replace_all("i|n","numeric") 
+          
+          muts_region = read.table(textConnection(""), col.names = maf_columns,colClasses = maf_types_sep)
+        }else{
+          muts_region = vroom::vroom(I(muts), col_types = paste(maf_column_types,collapse=""),
```

Comment:
Can this also be condensed to  `muts_region = vroom::vroom(I(muts),col_types = maf_column_types` as you did for line 2139 above?